src/Tizen.Applications.WatchfaceComplication/Tizen.Applications/EditablesContainer.cs

```diff
@@ -0,0 +1,348 @@
+            Log.Debug(_logTag, "request edit");
+            ComplicationError ret;
+
+            Interop.WatchfaceComplication.GetEditableContainer(out _container);
+            if (_container == IntPtr.Zero)
+            {
+                ErrorFactory.ThrowException(ComplicationError.EditNotReady, "Editor not ready");
+            }
+
+            if (_editableUpdatedCallback == null)
+                _editableUpdatedCallback = new Interop.WatchfaceComplication.EditableUpdateRequestedCallback(EditableUpdatedCallback);
+
+            foreach (Complication comp in _compList)
+            {
+                IEditable e = comp;
+                IntPtr hi = IntPtr.Zero;
+                if (e.Highlight != null && e.Highlight.Raw != IntPtr.Zero)
+                    hi = e.Highlight.Raw;
+                Interop.WatchfaceComplication.AddComplication(_container, e.EditableId, comp._handle, hi);
+            }
+
+            foreach (DesignElement de in _deList)
+            {
+                IEditable e = de;
+                IntPtr candidates;
+                Interop.WatchfaceComplication.CreateCandidatesList(out candidates);
+                foreach (Bundle b in de.Candidates)
+                {
+                    Interop.WatchfaceComplication.AddCandidatesListItem(candidates, b.SafeBundleHandle);
+                }
+
+                IntPtr hi = IntPtr.Zero;
+                if (e.Highlight != null && e.Highlight.Raw != IntPtr.Zero)
+                    hi = e.Highlight.Raw;
+                Interop.WatchfaceComplication.AddDesignElement(_container, e.EditableId, e.GetCurrentDataIndex(), candidates, hi, e.Name);
+                Log.Debug(_logTag, "Add design element done :" + e.Name);
+            }
+
+            ret = Interop.WatchfaceComplication.RequestEdit(_container, _editableUpdatedCallback, IntPtr.Zero);
+            if (ret != ComplicationError.None)
+            {
+                ErrorFactory.ThrowException(ret, "Request edit fail");
+            }
+        }
+
+        /// <summary>
+        /// Overrides this method if you want to handle the behavior when the editor is ready to edit.
+        /// </summary>
+        /// <param name="editorId">The appid of ready to edit editor.</param>
+        /// <since_tizen> 6 </since_tizen>
+        protected abstract void OnEditReady(string editorId);
+
+        /// <summary>
+        /// Loads the editable's current data.
+        /// </summary>
+        /// <returns>The editable's latest data that selected by editor.</returns>
+        /// <exception cref="ArgumentException">Thrown when some parameter are invalid.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method failed due to invalid operation.</exception>
+        /// <exception cref="NotSupportedException">Thrown when the watchface complication is not supported.</exception>
+        /// <remarks>
+        /// This API loads editable's last current data which is updated by editor application.
```

Comment:
This method loads the last editable data which is updated by editor application.